### PR TITLE
MINOR: JavaDoc improvements for RangeAssignor

### PR DESCRIPTION
**Pull request by astubbs**

### CODE REVIEW

**Comment by omkreddy** [comment]
LGTM

File: clients/src/main/java/org/apache/kafka/clients/consumer/RangeAssignor.java [hunk 1]
```diff
@@ -26,17 +26,19 @@
...
- * The range assignor works on a per-topic basis. For each topic, we lay out the available partitions in numeric order
+ * <p>The range assignor works on a per-topic basis. For each topic, we lay out the available partitions in numeric order
  * and the consumers in lexicographic order. We then divide the number of partitions by the total number of
  * consumers to determine the number of partitions to assign to each consumer. If it does not evenly
  * divide, then the first few consumers will have one extra partition.
  *
- * For example, suppose there are two consumers C0 and C1, two topics t0 and t1, and each topic has 3 partitions,
- * resulting in partitions t0p0, t0p1, t0p2, t1p0, t1p1, and t1p2.
+ * <p>For example, suppose there are two consumers <code>C0</code> and <code>C1</code>, two topics <code>t0</code> and <code>t1</code>, and each topic has 3 partitions,
+ * resulting in partitions <code>t0p0, t0p1</code>, <code>t0p2</code>, <code>t1p0</code>, <code>t1p1</code>, and <code>t1p2</code>.
```
Comment (astubbs) [inline clients/src/main/java/org/apache/kafka/clients/consumer/RangeAssignor.java]: Ah this one accidendtly includes the comma, I’ll fix that and push again.

**Comment by astubbs** [comment]
Ok, fixed it, check it out now.

**Comment by astubbs** [comment]
retest this please

**Comment by astubbs** [comment]
@omkreddy I fixed the typo (missing code wrappers around the ','), but I amended instead of just committing a new commit (sorry). I also cherry picked an old typo from another pr I made ages ago but never got merged.

**Comment by astubbs** [comment]
retest this please

**Comment by asfgit** [comment]
FAILURE
 7973 tests run, 5 skipped, 1 failed.
--none--

**Comment by asfgit** [comment]
FAILURE
 7973 tests run, 5 skipped, 1 failed.
--none--

**Comment by asfgit** [comment]
SUCCESS 
 8081 tests run, 5 skipped, 0 failed.
--none--

**Pull request merged**